src/Configuration/ConfigurationAnalyzer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Qossmic\Deptrac\Configuration;
+
+final class ConfigurationAnalyzer
```

Comment:
Shouldn't it be `AnalyzerConfiguration` instead?

Reply:
This naming is consistent with other specialized `Configuration*` classes like `ConfigurationRuleset`, `ConfigurationCollector` etc.